modules/contact/src/constraints/ComputeWeightedGapLMMechanicalContact.C

```diff
@@ -111,7 +114,7 @@ ComputeWeightedGapLMMechanicalContact::residualSetup()
 void
 ComputeWeightedGapLMMechanicalContact::jacobianSetup()
 {
-  residualSetup();
+  ComputeWeightedGapLMMechanicalContact::residualSetup();
```

Comment:
Haha this is a bit ridiculous but I understand why you must have it

Reply:
I changed my mind ... we shouldn't need this. You already override `jacobianSetup` in your derived and it calls `residualSetup` ... which will call the base `residualSetup` and then do your additional work in the derived `residualSetup`.

If a user, however, instantiates the base class then the `jacobianSetup` here will call the base `residualSetup` and everything will be hunky dory. So if somehow there *is* a need for this change, then something is wrong